packages/app/src/cli/models/app/app.ts

```diff
@@ -19,11 +19,11 @@ const LegacyAppSchema = zod
 const AppSchema = zod
   .object({
     name: zod.string(),
-    api_contact_email: zod.string(),
     client_id: zod.string(),
     scopes: zod.string().optional(),
-    webhook_api_version: zod.string(),
-    application_url: zod.string(),
+    api_contact_email: zod.string().optional(),
+    webhook_api_version: zod.string().optional(),
+    application_url: zod.string().optional(),
```

Comment:
Hey @Arkham, what's the reason for making these optional?

Reply:
we'd need to pass them everywhere, even in tests. what makes it super annoying is that if you forget to pass one of those the test will fail with no indication that it was the cause of failure.

Reply:
But if they are required fields as part of app config upstream, should that not be reflected?

Reply:
are they required? I thought all fields were optional except for `client_id`

Reply:
let's address this in a follow-up PR, I just wanted to quickly unblock myself :)